ui-src/package.json

```diff
@@ -0,0 +1,55 @@
+{
+  "name": "automated-perf-test",
+  "version": "0.0.0",
+  "license": "MIT",
+  "scripts": {
+    "ng": "ng",
+    "build": "ng build --prod",
+    "watch": "ng build --watch",
+    "test": "ng test",
+    "lint": "ng lint",
+    "e2e": "ng e2e"
+  },
+  "private": true,
+  "dependencies": {
+    "@angular/animations": "^5.1.3",
+    "@angular/cdk": "^5.0.3",
+    "@angular/common": "^5.0.0",
+    "@angular/compiler": "^5.0.0",
+    "@angular/core": "^5.0.0",
+    "@angular/forms": "^5.0.0",
+    "@angular/http": "^5.0.0",
+    "@angular/material": "^5.0.3",
+    "@angular/platform-browser": "^5.0.0",
+    "@angular/platform-browser-dynamic": "^5.0.0",
+    "@angular/router": "^5.0.0",
+    "bootstrap": "^3.3.7",
```

Comment:
Can we update the .angular-cli pointing to the dist in this folder. Somthing like this
     "../node_modules/@angular/material/prebuilt-themes/indigo-pink.css" to      "../node_modules/bootstrap/dist/bootstrap.min.css"
Then remove the cdn link in the index.html page.